doc/ref/configuration/master.rst

```diff
@@ -1520,6 +1520,77 @@ Jinja environment init variable ``lstrip_blocks``.
 
     jinja_lstrip_blocks: False
 
+.. conf_master:: jinja_comment_prefix
+
+``jinja_line_comment_prefix``
+-----------------------
```

Comment:
Please extend the dashed line so it is the same length as the line above. This reduces build warnings when we build the HTML docs.